cloud/linode/service_controller.go

```diff
@@ -110,5 +110,5 @@ func (s *serviceController) processNextDeletion() bool {
 func (s *serviceController) handleServiceDeleted(service *v1.Service) error {
 	klog.Infof("ServiceController handling service (%s) deletion", getServiceNn(service))
 	clusterName := strings.TrimPrefix(service.Namespace, "kube-system-")
-	return s.loadbalancers.EnsureLoadBalancerDeleted(context.Background(), clusterName, service)
+	return s.loadbalancers.EnsureLoadBalancerDeleted(context.TODO(), clusterName, service)
```

Comment:
Yes, it was. I noticed that we only use `context.Background()` in tests and all other non-test `context` usages were `context.TODO()`. It's not a hill I am going to die on, was just trying to standardize since I was looking at this part of the code 😄

Reply:
I mostly ask since we recently added a change on the CCM that included removing an instance of using `context.TODO`: https://github.com/linode/linode-cloud-controller-manager/pull/241/files
In CAPL we stick to `context.Background`.
I think it's worth standardizing on one or the other but not in this PR.

Reply:
Agreed. I will go ahead and revert it back